includes/class-wcpdf-main.php

```diff
@@ -913,6 +920,40 @@ public function set_phpmailer_validator( $mailArray ) {
 		return $mailArray;
 	}
 
+	/**
+	 * Logs the document creation to the order notes
+	 */
+	public function log_document_creation_to_order_notes( $document, $context ) {
+		if( ! empty( $document ) && ! empty( $context ) && isset( WPO_WCPDF()->settings->debug_settings['log_to_order_notes'] ) ) {
+			$message = __( 'PDF %s created on %s action.', 'woocommerce-pdf-invoices-packing-slips' );
+
+			// bulk document
+			if( $document->is_bulk ) {
```

Comment:
This will generate error notices because it will only be set on bulk documents. `isset( $document->is_bulk )` will probably avoid this (but please double check, I'm never sure between isset and !empty for these cases!)